### PR TITLE
Decouple the client form from the integration form

### PR DESCRIPTION
**Pull request by Vincent-Carrier**
This is gonna reset the width of the modal to what it was before your changes but I'll fix it later.

### CODE REVIEW

File: components/Button.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import classNames from "classnames";
 
 export default ({ onClick, children, outline, primary, secondary, danger }) => {
   const classes = classNames(
-    "inline-block cursor-pointer font-normal text-sm rounded p-2 m-1",
+    "inline-block cursor-pointer font-bold text-sm rounded p-2 m-1",
```
Comment (jaecSolutions) [inline components/Button.jsx:5]: back to font-normal

File: components/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          {children}
+        </div>
+      </div>
+      <style jsx>{`
+        .bg-darkened {
+          background-color: rgba(0, 0, 0, 0.4);
+        }
+
+        .modal {
+          max-height: 90vh;
```
Comment (jaecSolutions) [inline components/Modal.jsx:30]: Again, we still have the same modal size for the 2 forms.
The whole point to have this refactor is the possibility to have difference sizes for 2 differents uses cases
- Try {company} -> small modal
- Connect {partner} -> larger modal
@Vincent-Carrier

Comment (Vincent-Carrier) [reply components/Modal.jsx:30]: Yeah this is gonna come in the next commit. Wanted to stick with refactoring only.

File: components/IntegrationDetails.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import Button from "./Button";
+import { useContext } from "react";
+import { SelectedIntegration } from "./SelectedIntegration";
+
+export default ({ integration }) => {
```
Comment (jaecSolutions) [inline components/IntegrationDetails.jsx:5]: 👍 Clean!

**Pull request merged**